sdk/communication/azure-communication-administration/README.md

```diff
@@ -1,6 +1,8 @@
 # Azure Communication Administration client library for Java
 
-Azure Communication Administration is used for managing users and tokens for Azure Communication Services.
```

Comment:
Since the package will be used for both Identity and Phone Numbers, we should keep the description about "managing users and tokens for Azure Communication Services".

Reply:
Sure. Let me take care of it